tests/ludwig/config_validation/test_checks.py

```diff
@@ -459,3 +459,28 @@ def test_check_qlora():
         "type": "lora",
     }
     ModelConfig.from_dict(config)
+
+
+def test_check_prompt_task_and_template():
+    config = {
+        "model_type": "llm",
+        "input_features": [
+            text_feature(name="test", column="test"),
+        ],
+        "output_features": [binary_feature()],
+        "combiner": {
+            "type": "tabnet",
+            "unknown_parameter_combiner": False,
+        },
+        TRAINER: {
+            "epochs": 1000,
+        },
+    }
+
+    ModelConfig.from_dict(config)
+
+    config["prompt"] = {"task": "Some task", "template": ""}
+    with pytest.raises(ConfigValidationError):
+        ModelConfig.from_dict(config)
+
+    config["prompt"] = {"task": "Some task", "template": "{__task__}"}
```

Comment:
Are there some more lines of code for this test to show success?

Reply:
still working on it